docs/community/instances.md

```diff
@@ -6,7 +6,6 @@ title: Community-run instances
 ## Official Servers
 
 * 🔗 https://meet.jit.si
-* 🔗 https://8x8.vc
```

Comment:
This is not a valid instance, at least you cannot setup a meeting from it, rather it redirects to "learn more" only.